scripts/repository.lic

```diff
@@ -1230,7 +1251,7 @@ elsif cmd[0] =~ /^list-?update[sd]$/i
     end
   }
   list.unshift(headers)
-  respond "\n#{format_list.call(list)}\n\n"
+  puts "\n#{format_list.call(list, 'list-updates')}\n\n"
```

Comment:
do NOT use `puts` as it has known issues with terminal based FEs. In order to send actual XML, change the `respond` to `_respond`